src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    public Square(String color, String name) {
+        setColor(color);
+        setName(name);
+
+    }

```

Comment:
Don't begin the class implementation with an empty line. Remove the empty line at the start of the class body.